demosplan/DemosPlanCoreBundle/Logic/Procedure/ProcedureService.php

```diff
@@ -1112,21 +1112,15 @@ public function updateProcedure($data)
     public function updateProcedureObject(Procedure $procedureToUpdate)
     {
         try {
+            // todo: refactor this method so that there are differences between sourceProcedure and updatedProcedure
```

Comment:
We don't allow new TODOs in the code.

Reply:
removed in [32a9944](https://github.com/demos-europe/demosplan-core/pull/1741/commits/32a9944df961a25bce58dce1761fc4e17782d668)